scripts/calculateKpi.ts

```diff
@@ -136,6 +151,18 @@ async function getArgs() {
     .option('datadir', {
       description: 'Directory to save data',
       default: 'rewards',
+    })
+    .option('use-redis', {
+      boolean: true,
+      implies: ['redis-port', 'redis-host'],
+    })
+    .option('redis-port', {
+      type: 'number',
+      default: 6379,
+    })
+    .option('redis-host', {
+      type: 'string',
+      default: '127.0.0.1',
```

Comment:
nit: I would recommend to use `--redis-connection redis://127.0.0.1:6379` string instead.
1 single argument to indicate the same info.

Reply:
oh yeah good idea! i'll update